File: gimpi/23-24/k1/02Poeni/2prijemni1.py
n = int(input())

poeni = 0

for i in range(n): # za svaki unos
    pokusaj = input()
    resenje = input()

    poz1 = -1
    poz2 = -1

    for j in range(5): # odredi poziciju pokusaja kandidata od 0 do 4
        if pokusaj[j] == 'x':
            poz1 = j

    for j in range(5): # odredi poziciju resenja
        if resenje[j] == 'x':
            poz2 = j

    if poz1 == poz2: # tacno!
        poeni += 4
    elif poz1 != -1 and poz1 != poz2: # ne
        poeni -= 1
    #else: # bez pokusaja
        #poeni += 0

najvise = n * 4
najmanje = n * -1

print(poeni)
print(najvise)
print(najmanje)

File: gimpi/23-24/k1/02Poeni/2prijemni1.cpp
#include <bits/stdc++.h>
using namespace std;

int main() { 
    int n; cin >> n;
    int bodovi = 0;
    for (int i = 0; i < n; i++) {
        string proba, resenje;
        cin >> proba >> resenje;

        int poz1 = -1, poz2 = -1;

        for (int j = 0; j < 5; j++)
            if ( proba[j] == 'x' ) poz1 = j;

        for (int j = 0; j < resenje.size(); j++)
            if ( resenje[j] == 'x' ) poz2 = j;
        
        if ( poz1 == poz2 ) bodovi += 4;
        else if ( (poz1 != -1) && (poz1 != poz2) ) bodovi -= 1; 
        }
    cout << bodovi << '\n' << 4 * n << '\n' << -n << '\n';

}

